lib/msf/ui/console/command_dispatcher/exploit.rb

```diff
@@ -268,67 +268,6 @@ def cmd_rexploit_help
 
   alias cmd_rerun_help cmd_rexploit_help
 
-  # Select a reasonable default payload and minimally configure it
```

Comment:
You'll need to update this to call the implementation at the new location, either here - or the previous call site